frontend/main.tsx

```diff
@@ -42,6 +47,18 @@ function Main() {
             <Route path="/login">
               <Login/>
             </Route>
+            <Route path="/genebanks">
+              <Genebanks/>
+            </Route>
+            <Route path="/genebank/:genebankId">
+              <Genebank/>
+            </Route>
+            <Route path="/herd/:herdId">
+              <Herd/>
+            </Route>
+            <Route path="/individual/:individualId">
+              <Individual/>
```

Comment:
I would prefer to forward the routing information to the subcomponents here as props earlier. I don't know if the routing lib  supports this out of the box but one could imagine an API like

```
<Route path="/individual/:id">
  {({id}) => <Individual id={id}/>}
</Route>
```
This way the name `individualId` does need to span several files. This is good because it is not type-checked. That is what I want to achieve.

Reply:
Another benefit of this which is even better is that the `Individual` component will be easier to test since you won't need to set up routing to test it.

Reply:
yeah, that would be way neater! I was honestly surprised that I had to maintain variable name as well, but I guess it's a react thing or something... I haven't found any hacky syntax for this though...